DynamicMeter.c

```diff
@@ -102,10 +102,11 @@ static void DynamicMeter_getUiName(const Meter* this, char* name, size_t length)
    if (meter) {
       const char* uiName = meter->caption;
       if (uiName) {
-         int len = strlen(uiName);
+         size_t len = strlen(uiName);
+         assert(len < 32);
```

Comment:
Given this is user-controlled via a config file, this should be a runtime check (DiD).

Reply:
What does the colon in this `uiName` string do? And may I replace this `strlen` call with `(size_t)(String_strchrnul(uiName, ':') - uiName)` ?